pages/passport/barriers/identify-a-barrier.tsx

```diff
@@ -121,22 +119,24 @@ const IdentifyABarrier: React.FunctionComponent = () => {
               label={intl.formatMessage({
                 defaultMessage: "Barrier description",
               })}
+              rules={{
+                required: intl.formatMessage(errorMessages.required),
+              }}
```

Comment:
Again, if the user has the option to either name and describe their own barrier, or use the worksheet to pick one, then the name and description inputs must be optional.